tests/poster_mock.py

```diff
@@ -37,23 +38,28 @@ class PosterServerMock(BaseHTTPRequestHandler):
         LOCATIONS_PATTERN = re.compile(r"/clients.getLocations")
         TABLES_PATTERN = re.compile(r"/clients.getTables")
         TOKEN_PATTERN = re.compile(r"/auth/access_token")
+        CUSTOMERS_PATTERN = re.compile(r"/clients.getClients")
 
         def do_GET(self):
             if re.search(self.LOCATIONS_PATTERN, self.path):
                 self.send("locations")
                 return
+
             if re.search(self.TABLES_PATTERN, self.path):
                 self.send("tables")
                 return
 
+            if re.search(self.CUSTOMERS_PATTERN, self.path):
+                self.send("customers")
+                return
```

Comment:
@vladarefiev Are there any unit tests for ``poster_mock.py`` itself? If so, can you add a case for the customers?

Reply:
@amihaiemil No, there are not. `poster_mock` is imported in `test_poster` and there partially we test it.